t_stress/test_ddos.py

```diff
@@ -0,0 +1,208 @@
+listen 80 proto=http;
+listen 443 proto=h2,https;
+
+cache 2;
+cache_fulfill * *;
+cache_methods GET HEAD;
+cache_ttl 3600;
+
+access_log on;
+keepalive_timeout 15;
+
+frang_limits {{
+    request_rate {int(RPC / 2)};
+    request_burst {int(RPC / 10)};
+    tcp_connection_rate {int(CONNS / 2)};
+    tcp_connection_burst {int(CONNS / 10)};
+    concurrent_tcp_connections {int(CONNS / 2)};
+    client_header_timeout 20;
+    client_body_timeout 10;
+    http_uri_len 1024;
+    http_hdr_len 256;
+    http_ct_required false;
+    http_ct_vals "text/plain" "text/html" "application/json" "application/xml";
+    http_header_chunk_cnt 10;
+    http_body_chunk_cnt 0;
+    http_resp_code_block 403 404 502 5 1;
+    http_method_override_allowed true;
+    http_methods head post put get;
+    http_strict_host_checking true;
+
+    ip_block off;
+}}
+
+# Allow only following characters in URI: %+,/a-zA-Z0-9&?:-.[]_=
+# These are tested with the WordPress admin panel.
+http_uri_brange 0x25 0x2b 0x2c 0x2f 0x41-0x5a 0x61-0x7a 0x30-0x39 0x26 0x3f 0x3a 0x2d 0x2e 0x5b 0x5d 0x5f 0x3d;
+
+health_stat 3* 4* 5*;
+health_stat_server 3* 4* 5*;
+
+block_action attack drop;
+block_action error reply;
+
+# Make WordPress to work over TLS.
+# See https://tempesta-tech.com/knowledge-base/WordPress-tips-and-tricks/
+req_hdr_add X-Forwarded-Proto "https";
+
+resp_hdr_set Strict-Transport-Security "max-age=31536000; includeSubDomains";
+
+# Remove the proxy header to mitigate the httpoxy vulnerability
+# See https://httpoxy.org/
+req_hdr_set Proxy;
+
+tls_certificate ${{tempesta_workdir}}/tempesta.crt;
+tls_certificate_key ${{tempesta_workdir}}/tempesta.key;
+tls_match_any_server_name;
+
+srv_group main {{server ${{server_ip}}:8000 conns_n=128;}}
+
+vhost tempesta-tech.com {{proxy_pass main;}}
+
+http_chain {{
+	# Redirect old URLs from the old static website
+	uri == "/index"		-> 301 = /;
+	uri == "/development-services" -> 301 = /network-security-performance-analysis;
+
+	# Disable PHP dynamic logic for caching
+	# See https://www.varnish-software.com/developers/tutorials/configuring-varnish-wordpress/
+	uri == "/wp-admin*" -> cache_disable;
+	uri == "/wp-comments-post.php*" -> cache_disable;
+
+	# RSS feed /comments/feed/ is cached as other resource for 1 hour,
+	# defined by the global cache_ttl policy.
+
+	# Proably outdated redirects
+	uri == "/index.html"	-> 301 = /;
+	uri == "/services"	-> 301 = /development-services;
+	uri == "/services.html"	-> 301 = /development-services;
+	uri == "/c++-services"	-> 301 = /development-services;
+	uri == "/company.html"	-> 301 = /company;
+	uri == "/blog/fast-programming-languages-c-c++-rust-assembly" -> 301 = /blog/fast-programming-languages-c-cpp-rust-assembly;
+
+	-> tempesta-tech.com;
+}}
+"""
+    }
+
+    backends = [{"id": "wordpress", "type": "lxc", "external_port": "8000"}]
+
+    proxies = []
+
+    interface = tf_cfg.cfg.get("Server", "aliases_interface")
+
+    def get_response(self, curl, uri: str) -> None:
```

Comment:
Style: Please don't name methods with prefix `get_` if it is not getter method. In case of `get_response` it's little bit confusing, because such name doesn't show what really method do. Whether it just returns the response or it makes request and then returns the response(actually in this case method doesn't return anything).